### PR TITLE
JD5JB-32

### PR DESCRIPTION
**Pull request by TomaszOciepa**
adding a description to the flat when writing to a file

### CODE REVIEW

File: web/src/main/java/com/infoshareacademy/jbusters/web/LoadDistricTransactionServlet.java [hunk 1]
```diff
@@ -22,6 +24,8 @@
 public class LoadDistricTransactionServlet extends HttpServlet {
 
     private static final String TEMPLATE_NAME = "load-district";
+    private static final Logger LOG = LoggerFactory.getLogger(ValuationServlet.class);
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/LoadDistricTransactionServlet.java]: zla nazwa loggera

File: web/src/main/java/com/infoshareacademy/jbusters/web/LoadDistricTransactionServlet.java [hunk 2]
```diff
@@ -43,8 +47,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
 
         try {
             template.process(model, out);
+            LOG.info("Loaded district list", districtsList.size());
         } catch (TemplateException e) {
             e.printStackTrace();
+            LOG.error("Unable to load district list", districtsList.size());
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/LoadDistricTransactionServlet.java]: konkatenuj stirngi albo uzyj {} w komunikacie

File: web/src/main/java/com/infoshareacademy/jbusters/web/LoadDistricTransactionServlet.java [hunk 3]
```diff
@@ -43,8 +47,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
 
         try {
             template.process(model, out);
+            LOG.info("Loaded district list", districtsList.size());
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/LoadDistricTransactionServlet.java]: konkatenuj stirngi albo uzyj {} w komunikacie

File: web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java [hunk 4]
```diff
@@ -50,24 +55,78 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
...
+        model.put("flat_area", newTransaction.getFlatArea());
+        model.put("level", newTransaction.getLevel());
+        model.put("parking_spot", newTransaction.getParkingSpot());
+        model.put("standard_level", newTransaction.getStandardLevel());
+        model.put("construction", newTransaction.getConstructionYearCategory());
+
+        try {
+            template.process(model, out);
+        } catch (TemplateException e) {
+            e.printStackTrace();
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java]: logger zamiast printStackTrave

File: web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java [hunk 5]
```diff
@@ -50,24 +55,78 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
...
+        String plik = "Zapisane";
+
+        Map<String, String> model = new HashMap<>();
+        model.put("price", plik);
 
         try {
             template.process(model, out);
+            LOG.info("Saved user transaction to file", path);
         } catch (TemplateException e) {
             e.printStackTrace();
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java]: usun printStackTrave skoro logujesz wyjatek

File: web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java [hunk 6]
```diff
@@ -50,24 +55,78 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
...
-        Map<String, Object> model = new HashMap<>();
-        model.put("cena", flatPrice);
+        String plik = "Zapisane";
+
+        Map<String, String> model = new HashMap<>();
+        model.put("price", plik);
 
         try {
             template.process(model, out);
+            LOG.info("Saved user transaction to file", path);
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java]: konkatenuj stirngi albo uzyj {} w komunikacie

File: web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java [hunk 7]
```diff
@@ -50,24 +55,78 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
...
         } catch (TemplateException e) {
             e.printStackTrace();
+            LOG.error("Failed to save user file to", path);
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java]: konkatenuj stirngi albo uzyj {} w komunikacie

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 8]
```diff
@@ -25,7 +25,7 @@
 
     private DataLoader dataLoader;
 
-    private Path pathToUserFile = Paths.get("data", "test.txt");
+    private Path pathuserFile = Paths.get("data", "test.txt");
```
Comment (MichalNowakowski) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: pathToUserFile

File: web/src/main/java/com/infoshareacademy/jbusters/web/LoadCityTransactionServlet.java [hunk 9]
```diff
@@ -24,7 +26,7 @@
 public class LoadCityTransactionServlet extends HttpServlet {
 
     private static final String TEMPLATE_NAME = "load-city";
-
+    private static final Logger LOG = LoggerFactory.getLogger(ValuationServlet.class);
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/LoadCityTransactionServlet.java]: zla nazwa loggera

File: web/src/main/java/com/infoshareacademy/jbusters/web/LoadCityTransactionServlet.java [hunk 10]
```diff
@@ -44,8 +46,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
 
         try {
             template.process(model, out);
+            LOG.info("Loaded city list of size {}", cities.size());
         } catch (TemplateException e) {
             e.printStackTrace();
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/LoadCityTransactionServlet.java]: wyrzuc print stack trace

File: web/src/main/java/com/infoshareacademy/jbusters/web/LoadOtherValuesTransactionServlet.java [hunk 11]
```diff
@@ -21,6 +23,7 @@
 public class LoadOtherValuesTransactionServlet extends HttpServlet {
 
     private static final String TEMPLATE_NAME = "load-other-values";
+    private static final Logger LOG = LoggerFactory.getLogger(ValuationServlet.class);
```
Comment (MichalNowakowski) [inline web/src/main/java/com/infoshareacademy/jbusters/web/LoadOtherValuesTransactionServlet.java]: zla nazwa loggera

File: web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java [hunk 12]
```diff
@@ -45,6 +45,7 @@
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
         resp.addHeader("Content-Type", "text/html; charset=utf-8");
 
+        Transaction newTransaction = new Transaction();
```
Comment (isacademyci) [inline web/src/main/java/com/infoshareacademy/jbusters/web/ValuationServlet.java]: MAJOR Rename "newTransaction" which hides the field declared at line 36. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AHiddenFieldCheck)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 13]
```diff
@@ -134,22 +134,26 @@ private void saveSession(Transaction newTransaction, Path pathToFile, boolean is
...
-                            " ::::::::::::::::::::::::::::\n" + userList.get(i).toStringNoPrice());
+        if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+            if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+
+                List<Transaction> userList = dataLoader.createFlatsListFromFile(pathToUserFile, true);
+
+                if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+                    if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+                        for (int i = 0; i < userList.size(); i++) {
+                            System.out.println("\n:: MIESZKANIE NR " + (i + 1) + " " +
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 14]
```diff
@@ -75,7 +75,7 @@ private void menuSwitch(int Choise) throws IOException {
                 break;
             }
             case 3: {
-                saveSession(newTransactionCreator.getNewTransaction(), pathisUserFile, true);
+                saveSession(newTransactionCreator.getNewTransaction(), pathToUserFile, true);
                 break;
             }
             case 4: {
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MINOR Extract this nested code block into a method. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1199)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 15]
```diff
@@ -203,7 +207,7 @@ public void saveTransaction(Transaction newTransaction, Path pathToFile, boolean
         } finally {
             out.close();
         }
-
+        
         ConsoleViewer.clearScreen();
         System.out.println(":: Twoja transakcja została zapisana do pliku ::\n");
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 16]
```diff
@@ -134,22 +134,26 @@ private void saveSession(Transaction newTransaction, Path pathToFile, boolean is
...
+                List<Transaction> userList = dataLoader.createFlatsListFromFile(pathToUserFile, true);
+
+                if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+                    if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+                        for (int i = 0; i < userList.size(); i++) {
+                            System.out.println("\n:: MIESZKANIE NR " + (i + 1) + " " +
+                                    userList.get(i).getTransactionName() +
+                                    " ::::::::::::::::::::::::::::\n" + userList.get(i).toStringNoPrice());
+                        }
+                        System.out.println("\nPodaj nr mieszkania, które chcesz załadować");
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 17]
```diff
@@ -203,7 +207,7 @@ public void saveTransaction(Transaction newTransaction, Path pathToFile, boolean
         } finally {
             out.close();
         }
-
+        
         ConsoleViewer.clearScreen();
         System.out.println(":: Twoja transakcja została zapisana do pliku ::\n");
         System.out.println("Nowy wpis: " + newTransaction.toStringNoPrice());
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 18]
```diff
@@ -134,22 +134,26 @@ private void saveSession(Transaction newTransaction, Path pathToFile, boolean is
...
+                        for (int i = 0; i < userList.size(); i++) {
+                            System.out.println("\n:: MIESZKANIE NR " + (i + 1) + " " +
+                                    userList.get(i).getTransactionName() +
+                                    " ::::::::::::::::::::::::::::\n" + userList.get(i).toStringNoPrice());
+                        }
+                        System.out.println("\nPodaj nr mieszkania, które chcesz załadować");
+                        int chosenFlat = consoleReader.readInt(1, userList.size());
+                        ConsoleViewer.clearScreen();
+                        newTransactionCreator.setNewTransaction(userList.get(chosenFlat - 1));
+                        System.out.println(":: Mieskzanie nr " + chosenFlat + " o nazwie " +
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MAJOR Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 19]
```diff
@@ -134,22 +134,26 @@ private void saveSession(Transaction newTransaction, Path pathToFile, boolean is
...
-
-            List<Transaction> userList = dataLoader.createFlatsListFromFile(pathisUserFile, true);
-
-            if (!dataLoader.createFlatsListFromFile(pathisUserFile, true).isEmpty()) {
-                for (int i = 0; i < userList.size(); i++) {
-                    System.out.println("\n:: MIESZKANIE NR " + (i + 1) + " " +
-                            userList.get(i).getTransactionName() +
-                            " ::::::::::::::::::::::::::::\n" + userList.get(i).toStringNoPrice());
+        if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+            if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MAJOR Merge this if statement with the enclosing one. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1066)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 20]
```diff
@@ -75,7 +75,7 @@ private void menuSwitch(int Choise) throws IOException {
                 break;
             }
             case 3: {
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MINOR Extract this nested code block into a method. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1199)

File: app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java [hunk 21]
```diff
@@ -134,22 +134,26 @@ private void saveSession(Transaction newTransaction, Path pathToFile, boolean is
...
-                    System.out.println("\n:: MIESZKANIE NR " + (i + 1) + " " +
-                            userList.get(i).getTransactionName() +
-                            " ::::::::::::::::::::::::::::\n" + userList.get(i).toStringNoPrice());
+        if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+            if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+
+                List<Transaction> userList = dataLoader.createFlatsListFromFile(pathToUserFile, true);
+
+                if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
+                    if (!dataLoader.createFlatsListFromFile(pathToUserFile, true).isEmpty()) {
```
Comment (isacademyci) [inline app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java]: MAJOR Merge this if statement with the enclosing one. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1066)

**Comment by isacademyci** [comment]
SonarQube analysis reported 35 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 25 major
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Menu.java#L65](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L65): Add a default case to this switch. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![MAJOR][MAJOR] [Menu.java#L19](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L19): Remove this unused "LOGGER" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)
1. ![MAJOR][MAJOR] [Menu.java#L42](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L42): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L50](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L50): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L51](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L51): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L68](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L68): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L69](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L69): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L100](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L100): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L108](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L108): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L109](https://github.com/infoshareacademy/jjdd5-jbusters/blob/64f67448a2a10f52d9700d7e654da5f210229a8f/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L109): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by isacademyci** [comment]
SonarQube analysis reported 32 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 22 major
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Menu.java#L65](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L65): Add a default case to this switch. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![MAJOR][MAJOR] [Menu.java#L19](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L19): Remove this unused "LOGGER" private field. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)
1. ![MAJOR][MAJOR] [Menu.java#L42](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L42): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L50](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L50): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L51](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L51): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L68](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L68): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L69](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L69): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L100](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L100): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L108](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L108): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)
1. ![MAJOR][MAJOR] [Menu.java#L109](https://github.com/infoshareacademy/jjdd5-jbusters/blob/702f34bcbb46ff8bb541fe1f9044a5d3b0f04147/app/src/main/java/com/infoshareacademy/jbusters/console/Menu.java#L109): Replace this use of System.out or System.err by a logger. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS106)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**